### ISSUE TITLE
Client dependencies are not listed 

### ISSUE THREAD

**Issue by raam86** [http4s/http4s#1137]
I couldn't find a place where I had all the dependencies for the http4s client was listed. 

the correct list is: 

```
val http4sV = "0.17.0-M1"
val http4s = Seq(
"org.http4s" %% "http4s-core" % http4sV, 
"org.http4s" %% "http4s-client" % http4sV,
"org.http4s" %% "http4s-blaze-client" % http4sV
)
```

**Comment by rossabaker** [comment]
There was some discussion of making a g8 template that includes the client.  I think I'm not too keen on maintaining another branch of g8, but that our client tut should show an sbt with the proper dependencies.

**Comment by ChristopherDavenport** [comment]
Closed by #1166
